plugins/inputs/win_perf_counters/README.md

```diff
@@ -92,6 +104,14 @@ This must follow before other plugin configurations,
 beneath the main win_perf_counters entry, `[[inputs.win_perf_counters]]`.
 
 Following this are 3 required key/value pairs and three optional parameters and their usage.
+#### Computers
```

Comment:
Set this option only at the top level of the plugin:
```
[[inputs.win_perf_counters]]
  computers = ["127.0.0.1", "SQL-SERVER-01"]
  [[inputs.win_perf_counters.objects]]
    object_name = "LogicalDisk"
```

This will be a bit less flexible but this is a reasonable level of control and more in line with expectations from other plugins.

Reply:
Make sense to have it global for all objects
But as some instances of the same performance object can have different name on different computers (e.g. network interfaces) as would allow to override the source at object level.

Reply:
This will be okay as the user can have multiple `[[inputs.win_perf_counters]]`.

Reply:
And what about possibility to override this at performance object level  `[[inputs.win_perf_counters.object]]` (currently left in the implementation)? Does it seems confusing?
It allows a bit more compact configuration.
When using  multiple `[[inputs.win_perf_counters]]`, user will have to copy all other global parameters and sync them every time a change is needed.